From 69f84f95fa875c6f562294b1a6a1ea6f584d3f6c Mon Sep 17 00:00:00 2001
From: Dieter Hametner <dh (plus) vdr (at) gekrumbel (dot) de>
Date: Sat, 4 May 2013 22:27:09 +0200
Subject: [PATCH 2/2] With tntnet v2.2 use also the request.getArg(<name>)
 function. In the previous commit support for tntnet 2.2 was added. The URL
 mapping changed in that version and allows now named arguments. This change
 makes uses of this feature now.

---
 pages/content.ecpp | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/pages/content.ecpp b/pages/content.ecpp
index 27d827c..cde092f 100644
--- a/pages/content.ecpp
+++ b/pages/content.ecpp
@@ -17,7 +17,11 @@ bool logged_in(false);
 
 string mime("image/png");
 if (request.getArgsCount() > 0) {
+#if TNT_MAPURL_NAMED_ARGS
+	mime = request.getArg("mime-type");
+#else
 	mime = request.getArg(0);
+#endif
 	// dsyslog("vdrlive::content found mime arg (%s)", mime.c_str());
 }
 reply.setContentType(mime);
-- 
1.8.3.2

